docs/docusaurus.config.js

```diff
@@ -0,0 +1,99 @@
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').DocusaurusConfig} */
+module.exports = {
+  title: 'Open Sauced',
+  tagline: 'Dinosaurs are cool',
+  url: 'https://docs.opensauced.pizza',
+  baseUrl: '/',
+  onBrokenLinks: 'throw',
+  onBrokenMarkdownLinks: 'warn',
+  favicon: 'img/favicon.ico',
+  organizationName: 'open-sauced', // Usually your GitHub org/user name.
+  projectName: 'open-sauced', // Usually your repo name.
+  trailingSlash: true,
+  themeConfig: {
+    navbar: {
+      logo: {
+        alt: 'My Site Logo',
+        src: 'img/logo.svg',
+      },
+      items: [
+        {
+          type: 'doc',
+          docId: 'introduction',
+          position: 'left',
+          label: 'Docs',
+        },
+        {
+          href: 'https://github.com/facebook/docusaurus',
```

Comment:
The GitHub link should probably lead to our repo page

```suggestion
          href: 'https://github.com/open-sauced/open-sauced',
```